tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -919,6 +923,9 @@ private void computeOne(int n) {
             if (progressCallback.isStopped()) {
                 throw new RuntimeException("Requested to stop");
             }
+            if (DEBUG) {
+                MemoryHelper.availableMemory("VettingViewer.WriteAction.computeOne", true);
+            }
```

Comment:
```suggestion
            MemoryHelper.availableMemory("VettingViewer.WriteAction.computeOne", DEBUG);
```

Reply:
minor, but this would cause the memory to be always written if there's a verbose log somewhere.

Reply:
if and when the return value is used, then you're right, DEBUG could replace "true"

in the meantime, the method, and the methods it calls, shouldn't be called when they have no effect -- that would waste time and possibly harm performance